app/config.json

```diff
@@ -0,0 +1,37 @@
+[
+	{
+		"id": 1,
```

Comment:
you really do not need ids for this or labels. this entire file is too verbose it should just look like this.
```
[
    {"Root", "path"},
    {"Globals", "path2"},
    {"Current Theme", "path3"},
    {"Template", "path4"},
    {"SCSS", "path5"}
]
```

but we can fix this after its working i suppose